src/services/profile.service.js

```diff
@@ -0,0 +1,89 @@
+import { ApiError } from '../exeptions/api.error.js';
+import { User } from '../models/user.js';
+import { emailService } from './email.service.js';
+import { v4 as uuidv4 } from 'uuid';
+import { jwtService } from './jwt.service.js';
+import bcrypt from 'bcrypt';
+
+function getAllInfo() {
+  return User.findAll({
+    where: {
+      activationToken: null,
+    },
+  });
+}
+
+function normalize({ id, name, email, password }) {
+  return {
+    id,
+    name,
+    email,
+    password,
+  };
+}
+
+function findByEmail(email) {
+  const user = User.findOne({ where: { email } });
+
+  return user;
+}
+
+async function register(name, email, password) {
+  const activationToken = uuidv4();
+
+  const existUser = await findByEmail(email);
+
+  if (existUser) {
+    throw ApiError.badRequest('User alredy exist', {
+      email: 'User alredy exist',
```

Comment:
There is a typo in the error message: 'User alredy exist'. It should be 'User already exists'.